docs/model_developers/modules.rst

```diff
@@ -167,14 +167,14 @@ The format for requires in requirements.txt is the folllowing:
 Plugins
 *******
 Plugins provide :ref:`functions<lang-plugins>` that can be called from the :term:`DSL`. This is the
-primary mechanism to interface Python code with the configuration model at compile time. This
-mechanism is also used std::template or std::file. Inmanta also registers all plugins with the
-template engine (Jinja2) to use as filter.
+primary mechanism to interface Python code with the configuration model at compile time. For Example,
+this mechanism is also used for std::template and std::file. Furthermore, Inmanta also registers all
```

Comment:
Furthermore is a difficult word for none-native speakers, best check with bart and/or stefan to see if we have a better word.

Reply:
@bartv, @swal input is welcome

Reply:
After feedack:
- changed "furthermore" to "in addition to this"
- changed all occurrences of "configuration model" to "orchestration model"

should be ready to be merged.